src/stores/map.ts

```diff
@@ -117,6 +120,64 @@ export const useMapStore = defineStore("map", () => {
     }
   }
 
+  /**
+   * Create manual clusters for a given set of geographic spaces.
+   * @param {FeatureCollection} geoJsonSpaces - A GeoJSON FeatureCollection of spaces.
+   * @returns {FeatureCollection} - A new FeatureCollection containing clustered and original features.
+   */
+  function createManualClusters(geoJsonSpaces: FeatureCollection): FeatureCollection {
+    // Group the features by building slug
+    const groupedSpaces = geoJsonSpaces.features.reduce<{ [key: string]: Feature[] }>(
+      (acc, space) => {
+        const buildingSlug = space.properties?.buildingSlug;
+        if (!acc[buildingSlug]) {
+          acc[buildingSlug] = [];
+        }
+        acc[buildingSlug].push(space);
+        return acc;
+      }, {});
+
+    const clusters = Object.entries(groupedSpaces).map(([buildingSlug, spaces]): Feature => {
+      // Calculate the cluster center by averaging the coordinates of all spaces in the cluster
+      const clusterCoordinates = spaces.reduce<[number, number]>(
+        ([sumLon, sumLat], space) => {
+          const coordinates = (space.geometry as Point).coordinates;
+          return [sumLon + coordinates[0], sumLat + coordinates[1]];
+        },
+        [0, 0]
+      );
+
+      const count = spaces.length;
+      const buildingOccupancy = spaces[0].properties?.buildingOccupancy;
+      const buildingSlugModified = spaces[0].properties?.buildingSlugModified;
+
+      // Create a cluster feature with the calculated center and properties
+      return {
+        type: "Feature",
+        properties: {
+          buildingSlugModified,
+          buildingSlug,
+          buildingOccupancy,
+          point_count: count,
+          point_count_abbreviated: count,
```

Comment:
Normally I would say yes when using the stock clustering functionality. it will automatically add `point_count` and  `point_count_abbreviated` when clustering. `point_count_abbreviated` is similar to `point_count` in that it represents the number of points in a cluster, but it is a shortened version of the value. It is typically used in cases where the number of points in a cluster is very large, and displaying the full value of point_count would take up too much space on the map. for ecample it shows 1k instead of 1000. but we are getting less spaces so `pointCount` will do it for now.